### ISSUE TITLE
WooCommerce cart items and amount issue

### ISSUE THREAD

**Issue by adnan-amjad26** [Automattic/amp-wp#1372]
Hi, I thought to use this beta version for my website and after placing an order through AfterPay when redirected back to my website, the order is generated but the cart items never get cleared, the cart first shows amount and items quantity as 0, but when the page is completely loaded the items amount comes back. After that when I go to cart by clicking the cart amount in the header, the items still stay in the cart.

Please fix this issue asap. Thanks.

**Comment by amedina** [comment]
The validation errors are because the WooCommerce plugin, and any extensions you may be using, and the theme are not AMP compatible. In the future, WooCommerce itself may be AMP compatible as well as the extensions and themes to be used with WooCommerce. At this moment, to achieve compatibility you need to do the engineering work on your side to make these components AMP compatible. Check the [Adding Theme Support](https://github.com/Automattic/amp-wp/wiki/Adding-Theme-Support), and [Implementing Interactivity](https://github.com/Automattic/amp-wp/wiki/Implementing-Interactivity) in the Wiki.

**Comment by westonruter** [comment]
> Can I add AMP support to only homepage, posts, product and shop pages of my website? I have done so but facing alot of AMP validation errors in Console after adding #development=1 in the url.

Yes you can. As of [1.0-beta2](https://github.com/Automattic/amp-wp/releases/tag/1.0-beta2). What that version of the plugin you can enable Native mode or Paired mode with auto-sanitization, and then just select the specific templates you want to serve in AMP. You can read more about this in the [1.0-beta1 release post](https://make.xwp.co/2018/07/06/amp-plugin-release-v1-0-beta1/) under “Toggles for Templates that Support AMP.”.
